ci/prow/update_all_boskos_permissions.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
```

Comment:
Also please update README.md with an entry for this new script.

Reply:
Thanks for pointing this out. Updated.